src/foremast/utils/slack.py

```diff
@@ -35,7 +35,7 @@ def post_slack_message(message, channel):
     LOG.debug('Slack Channel: %s\nSlack Message: %s', channel, message)
     slack = slacker.Slacker(SLACK_TOKEN)
     try:
-        slack.chat.post_message(channel, message)
+        slack.chat.post_message(channel=channel, text=message, username='pipeline-bot', icon_emoji=':gear:')
```

Comment:
Make `username` and `icon_emoji` arguments for `post_slack_message()` so we, or other people, can change a little more easily.